src/components/ShareButton.tsx

```diff
@@ -83,7 +83,7 @@ export const ShareButton = memo(function ShareButton() {
     billingContext.tier === undefined
 
   return (
-    <Popover className="relative flex">
+    <Popover className="relative hidden sm:flex">
```

Comment:
Hey I think this PR pretty much removes the need for https://github.com/KittyCAD/modeling-app/pull/6776 no?

Reply:
Yeah @pierremtb I think a replacement for #6776 needs to be made with this top bar in mind. I'll chat with you in Slack about it. Since this PR this is what a mobile-dimensioned browser view looks like:

<img width="936" height="1768" alt="Screenshot 2025-07-14 at 10 34 52 AM" src="https://github.com/user-attachments/assets/ff406a91-259f-4edd-8924-03df9dc1caaa" />

Definitely still some cleanup to be done, but we might have an easier time with the toolbar like we can just make it scroll on mobile.